config/webpack.config.dev.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const webpack = require('webpack')
+
+module.exports = {
+  devtool: '#source-map',
+  externals: ['cozy'],
+  module: {
+    loaders: [{
+      test: require.resolve('cozy-bar/dist/cozy-bar.js'),
+      loader: 'imports?css=./cozy-bar.css'
+    }]
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      __SERVER__: JSON.stringify('http://app.cozy.local'),
```

Comment:
Other though here : will it work with the new mechanism of `cozy-stack serve --appdirs` https://github.com/cozy/cozy-stack/blob/master/cmd/serve.go#L35, like on a locally served `http://files.cozy.local` for example ?
ping @nono